### ISSUE TITLE
How to let Batchnorm to use global mean in both training and inferance?

### ISSUE THREAD

**Issue by saturdays** [pytorch/pytorch#15920]
## ❓ Questions and Help
Problem: If I use the default bn layer and use model.eval() before infence.  Test result has a significant discrepancy between test and other test.
Guess: I guess this is because bn layer uses different mean and var in different tests. And model is senstive to these mean/var changes?
Question: (1) Is My guesses correct? If correct, I can I always use global mean and global var so that the result is same in both training and eval? (2) Is these bn parameters designed to changes with time? So different inference sequence causes diffferent result?
Tried: I tried set momentum to 0 or track_running_stats=False after load the model. Both doesn't work.

### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by saturdays** [comment]
What's the good to keep bn parameters changing with time?

**Comment by saturdays** [comment]
If a set momentum to 0.0 and 0.1 in different tests. The results are also different. Is this discrepancy designed to be True?
